src/cryptonote_core/service_node_rules.cpp

```diff
@@ -280,4 +280,18 @@ bool get_portions_from_percent_str(std::string cut_str, uint64_t& portions) {
   return false;
 }
 
+uint64_t get_atomic_amount_from_percent_str(std::string cut_str) {
```

Comment:
This function should probably be renamed; HF19 registrations are 0-10000 integer values, so I guess they are "atomic", but don't want to confuse it with atomic oxen.  These is, I believe, coincidentally the same as financial basis points, so perhaps "percent_to_basis_points" would work?